tests/bluima_dummymodelproxy/pom.xml

```diff
@@ -0,0 +1,60 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
I tried putting everything in model proxy but couldn't make it work -- maybe the file was not properly packaged or not in the correct jar, I don't know exactly. I can rework on that later next week if needed.

In the meantime I'll move the folder.